src/main/java/ezschedule/model/event/UniqueEventList.java

```diff
@@ -50,9 +50,9 @@ public void add(Event toAdd) {
     }
 
     /**
-     * Replaces the person {@code target} in the list with {@code editedPerson}.
+     * Replaces the event {@code target} in the list with {@code editedPerson}.
      * {@code target} must exist in the list.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the list.
+     * The event identity of {@code editedPerson} must not be the same as another existing event in the list.
```

Comment:
`{@code editPerson}` should be changed too!

Reply:
Fixed